doc/developers/maintainer.rst

```diff
@@ -234,7 +228,13 @@ Making a release
    to the real https://pypi.org (replace "testpypi" by "pypi" in the "Run
    workflow" form).
 
-7. Alternatively, it's possible to collect locally the generated binary wheel
+7. Confirm that the bot has detected the tag on the conda-forge feedstock repo:
```

Comment:
Can this step 6?

Then "Trigger the GitHub Actions..." will be step 7 and then "Alternatively, it's possible to collect the generated binary..." is 7.1, since it is an alternative.